lib/bolt/plugin/env_var.rb

```diff
@@ -31,7 +32,8 @@ def validate_resolve_reference(opts)
       end
 
       def resolve_reference(opts)
-        ENV[opts['var']] || opts['default']
+        reference = ENV[opts['var']] || opts['default']
```

Comment:
I don't think we want to parse the default. If you want structured data as the default, you should just pass structured data as the value, not a JSON-encoded string version of it.